src/lua/luaTask.c

```diff
@@ -207,18 +213,35 @@ static bool _load_script(void)
         return true;
 }
 
+static bool user_bypass_requested(void)
+{
+        pr_info("lua: Checking for Lua runtime bypass request\r\n");
+        bool bypass = false;
+        LED_disable(ERROR_LED);
```

Comment:
this was to guarantee consistent flashing (first toggle enables LED). the function won't know the state of the LED when entered.